test/e2e/multicast_test.go

```diff
@@ -0,0 +1,328 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+	"sync"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	"antrea.io/antrea/pkg/features"
+)
+
+func skipIfMulticastDisabled(tb testing.TB) {
+	skipIfFeatureDisabled(tb, features.Multicast, true, false)
+}
+
+var nodeMulticastInterfaces [][]string
+var transportInterface string
```

Comment:
avoid to define such gloabl variables, it should not be hard to pass them via arugments. They are easy to conflict in this package.

Reply:
updated. removed